src/test/java/pages/InProduct.java

```diff
@@ -24,6 +24,9 @@
     private WebElement continueShopping;
     private final String endpoint = "index.php?id_product=7&controller=product";
```

Comment:
the same, as i understand, you relay on const product, that's not a best way of testing